### ISSUE TITLE
available for MCPE

### ISSUE THREAD

**Issue by DjJakkiGlass** [JacksonML/PrisonMining#1]
I run a mcpe server, and we want this plugin so bad! When will you dev it  for PE?
Otherwise, I have no clue how we are gonna do this. Any suggestions? Updates? Please!
Thanks :)

**Comment by JacksonML** [comment]
If you would like it I can try to start working on it again. When would you need it by?

**Comment by DjJakkiGlass** [comment]
Really? That would be so amazing!
Ummmm....we were hoping to open our server to the public within the next
week, but if you need more time we are flexible :) I honestly don't know
how long it takes to write a plugin so....lol. Let me know how long u need
and we can work from there.
Thanks!

**Comment by JacksonML** [comment]
Its hard to say, I don't know what I was doing when I wrote it, but currently the plugin is broken. I want to rewrite a fair bit of it. Do you have a specific date next week?

**Comment by DjJakkiGlass** [comment]
No specific date. My team is still building stuff and there is still a lot
of little things to tweak and test. I wanted to have it open by or before
Aug. 1st. So in about 10 days. Will that work for u? Remember, we are
flexible so don't feel bad if it takes longer :)
You are the one doing us the favor and we appreciate you!

**Comment by JacksonML** [comment]
That might work. How big are the mines that you will be resetting? I'm trying to spend time on certain features, and knowing your needs will help me spend more time on the right features. How many mines do you think you'll have?

**Comment by DjJakkiGlass** [comment]
About 10 mines. Maybe 10 blocks deep, wide, and long. I'm thinking I will
have between 5-10 players in at a time. I will also be using mine
reset...unless you can add that feature or config it to work with  mine
reset plugin.

Thanks!

**Comment by JacksonML** [comment]
If you will be using MineReset then why do you need my plugin? My plugin does essentially what MineReset does except MineReset has been in development longer than mine has.

**Comment by DjJakkiGlass** [comment]
Won't I need a specific plugin that has a command to send ppl to jail, and
to not let them leave until they mine a certain amount of ores? Plus, don't
you want your plugin to be featured and credited on my server?
I assumed prison mining was more of a minigame than anything.....

**Comment by JacksonML** [comment]
Ooh ok. Thats actually a good idea. I mean I definitely would like to have it featured but I didn't know that is what you wanted. Alright cool I'll see what I can do!

**Comment by DjJakkiGlass** [comment]
Awesome :)
Yeah man I'm trying to set my server apart from everyone else. If u wanna
come see what we have so far, I can put u on my whitelist. What's your
player name?
Here's the IP: UtopiaCraft.mcfrag.net
                   Port: 19132
Thanks!

**Comment by JacksonML** [comment]
JacksonML, just like my GitHub name

**Comment by DjJakkiGlass** [comment]
Cool! I will add you :)

**Comment by DjJakkiGlass** [comment]
U are in! Come by whenever you want. I'm on and off all day :)

**Comment by JacksonML** [comment]
Hey it appears that it is offline. It was online when I was not on the whitelist, but once you added me the server is no longer online.

**Comment by DjJakkiGlass** [comment]
Ikr? U broke my server, lol jk. I am having serious issues with PurPerms :(

**Comment by DjJakkiGlass** [comment]
It's OK now if u want me to give u a tour :)

**Comment by DjJakkiGlass** [comment]
So what do u think about the server?
I have an interesting question for u:
Would it be possible to add a feature to the PM game that allows staff
members to touch a person and send them to a jail cell? Like a command when
implemented would allow me or an admin hit a player and that action sends
them to a jail cell?
Also, QueenBey809 wanted you to come back to the server if you can.

Thanks!

**Comment by DegenApeDev** [comment]
I also run kidzzone.ca think we could use this on one of our minigame servers??

**Comment by DjJakkiGlass** [comment]
Awww I love that server! So cute! Sure :) that would be fun for the kids
there to play!
I also need help with other minigames. I have CTF and spleef installed, but
idk what to do? Am I supposed to build the area and then create a world? I
just have no clue how to set these things up :( I need serious help, lol.

**Comment by JacksonML** [comment]
I'm fine with anyone running my plugin, linuzo. Jakki, I could, however that may come in an update if you are still planning on releasing this week. /jail is the current command for jailing a player.

**Comment by DjJakkiGlass** [comment]
Thanks Jackson!

**Comment by DjJakkiGlass** [comment]
Hi jackson,
Have you finished the plugin yet?

**Comment by JacksonML** [comment]
No, I'm still working on it. School has slammed me with work so it's been hard to even work on personal projects. Sorry for the wait guys.

**Comment by DjJakkiGlass** [comment]
Dont worry bout it. Just hadnt heard from u in a while. School is a
priority. Just let me know whats up when u can 😊

**Comment by DjJakkiGlass** [comment]
Hi Jackson!
Hope school is going well =)
Having a lot of serious  issues with Cookedporkchop... The servers host?
Well the servers been down for 5 days... And I just wanted to let u know
its still cool if u need to take your time with the prison mining plugin.
Once the issues get resolved with CPC.. I'm moving the server to a new
hosting company.. U wouldn't  happen to know how I could go about getting
my files from them would u?
I've already requested they send them to me but... No response :/
So,  reply when u can,  hope all is well!

**Comment by uselesswaifu** [comment]
@DjJakkiGlass lol
